.github/workflows/test.yml

```diff
@@ -2,8 +2,6 @@ name: Tests
 on:
   pull_request:
   push:
-    branches:
```

Comment:
All PRs are build and pushes to `master` as well. What's the reason to build CI for all pushes? :thinking:

Reply:
🆗